OVERVIEW.md

```diff
@@ -0,0 +1,123 @@
+
+
+
+### Repo Structure
+```
+📂 hydra-torch
+└ 📁 configen 				# source files used when generating package content. all projects have their own subdirectory here.
+└ 📁 hydra-configs-torch		# a project corresponding to configuring a specific library (contains package definition)
+└ 📁 hydra-configs-torchvision		# "
+└ 📁 hydra-configs-<future-library> 	# "
+└ 📁 examples				# use-cases and tutorials
+```
+
+Each `hydra-configs-<library-name>` defines its own package corresponding to a project it provides classes for. That is, `hydra-configs-torch` contains a package (with its own `setup.py`) which provides classes for `torch`.
+
+[`hydra-configs-projects.txt`](hydra-configs-projects.txt) specifies a list of projects this repo maintains. When adding a project, please update this file.
+
+
+### Packaging
+
+This repo maintains multiple packages. An important area of consideration for these packages is to organize them such that they are compatible with each other and have an 'expected', intuitive structure. The easiest way to do this is to enforce a standard for all Hydra configs packages to follow.
+
+Our approach makes use of [Native Namespace Packages](https://packaging.python.org/guides/packaging-namespace-packages/#native-namespace-packages). This allows all packages to share the top level namespace `hydra_configs`. The only real requirement for this package format is to ensure there is no `__init__.py` located in the namespace folder.
+
+The format for folder structure per project is always:
+```
+📂 hydra-configs-<library-name>     # the dir containing the package of configs for <library>
+├ 📁 hydra_configs		    # the namespace we will always use
+│ └ 📁 <library-name>		    # e.g. 'torchvision'
+│   └ 📁 <library-subpackage-name>  # e.g. 'transforms'
+│      ⋮
+│      └ <module>.py                # e.g. 'transforms.py'
+└ setup.py 			    # configures this package for setuptools
+```
+
+The beauty of this approach is that users can be sure the importing idiom is reliably:
+```python
+from hydra_configs.<project_name>.path.to.module import <ClassName>Conf
+```
+
+This will retain compatibility even with repositories that define configs for hydra outside of the `hydra-torch` repo as long as they follow this pattern.
+
+##### Metapackage
+
+For convenience, we also provide a metapackage called `hydra-torch`. This is defined at the top level of the repo in `setup.py`. Effectively, all this means is that a user can either install each project-specific package as needed or they can obtain them all in one shot by simply installing this singular metapackage.
+
+### Configen
+
+We are actively developing the tool, [Configen](https://github.com/facebookresearch/hydra/tree/master/tools/configen) to automatically create config classes for Hydra. Much of the work for `hydra-torch` has helped prototype this workflow and it is still rapidly evolving.
```

Comment:
From the perspective of users of this repo, they don't care that configen is evolving and that this repo is helping guiding its evolution.

```suggestion
[Configen](https://github.com/facebookresearch/hydra/tree/master/tools/configen) is a tool used to generate config classes for Hydra. This repo relies on it to generate and maintain most of the config dataclasses.
```

Reply:
Looks like you didn't use this feedback in your CONTRIBUTING file.